crates/polars-core/src/chunked_array/logical/categorical/stringcache.rs

```diff
@@ -16,6 +15,7 @@ use crate::prelude::InitHashMaps;
 /// to determine how many threads use the string cache
 /// if the refcount is zero, we may clear the string cache.
 pub(crate) static USE_STRING_CACHE: AtomicU32 = AtomicU32::new(0);
+static STRING_CACHE_UUID_CTR: AtomicU64 = AtomicU64::new(0);
```

Comment:
Nice solution!

Reply:
I don't know how/when it's used enough to give meaningful feedback. The original was a `u128` so I wanted to be conservative with how much I shrink it.

Reply:
Yes, that was a naive solution because I just took time.

We will never have 4 billion unique string cache inits, so u32 will be large enough.